src/sql/workbench/contrib/notebook/test/browser/notebookActions.test.ts

```diff
@@ -347,6 +347,51 @@ suite('Notebook Actions', function (): void {
 		assert.strictEqual(actualMsg, expectedMsg);
 	});
 
+	test('Run with Parameters Action with no empty parameter cell in notebook', async function (): Promise<void> {
+		const testContents: azdata.nb.INotebookContents = {
+			cells: [{
+				cell_type: CellTypes.Code,
+				source: ['\n'],
+				metadata: { language: 'python' },
+				execution_count: 1
+			}],
+			metadata: {
+				kernelspec: {
+					name: 'python',
+					language: 'python',
+					display_name: 'Python 3'
+				}
+			},
+			nbformat: 4,
+			nbformat_minor: 5
+		};
+		let expectedMsg: string = localize('emptyParametersCell', "This notebook cannot run with parameters until there are parameters added to the parameter cell.[Learn more](https://docs.microsoft.com/sql/azure-data-studio/notebooks/notebooks-parameterization).");
```

Comment:
When making tests that verify strings it's best if you don't just copy the strings since then it makes it more annoying to have to remember to update this one as well if the actual string changes in the future. Just use the original string directly.